### ISSUE TITLE
0.4.3.131 and check_files - "No drives found" when no matches

### ISSUE THREAD

**Issue by box293** [mickem/nscp#126]
When using check_files, when it can't find any files the default message is "No drives found".

```
Command:
check_nrpe -H E4300 -c check_files -a path='C:\\No Backup' pattern=*.* 'filter=size gt 5000M' 'critical=count>0' 'top-syntax={${count} Files Found} ${list}' 'detail-syntax=%(file): %(size)'

Output:
No drives found|'count'=0;0;0
```
Exit code of 0 is returned.

```
Command:
check_nrpe -H E4300 -c check_files -a file='C:\\No Backup\\file_does_not_exist.iso' max-depth=1 'critical=count<1'

Output:
No drives found|'count'=0;0;1
```
Exit code of 2 is returned.

**Comment by mickem** [comment]
This has been fixed some time ago actually...

Regarding the exit code it is configurable using the empty-state option

**Comment by box293** [comment]
Thanks for the info about the empty-state option, works well.

I went back and tested and I found that this appears to be fixed in 0.4.3.138, it reported "No files found" instead of "No drives found" ... helpful if anyone needed to know in the future.

**Comment by mickem** [comment]
thanks for reporting back...
